src/utils/functions.util.js

```diff
@@ -0,0 +1,9 @@
+import {jwtTokens} from "../database/models/index"
+import { invalidateTokens } from "../services/token.service";
+export const checkRevoked=async(token)=>{
+    const tokenData=await jwtTokens.findOne({ where: { token: token } });
+    if(tokenData.dataValues.revoked===true){
+        return true;
+    }
+    return false;
+}
```

Comment:
- Don't use dataValues 'cause it's not necessary. Ask Yves for help
- Rename this file